crates/webidl/src/first_pass.rs

```diff
@@ -690,6 +691,25 @@ impl<'src> FirstPass<'src, ()> for weedle::CallbackDefinition<'src> {
     }
 }
 
+impl<'src> FirstPass<'src, ()> for weedle::CallbackInterfaceDefinition<'src> {
+    fn first_pass(&'src self, record: &mut FirstPassRecord<'src>, _: ()) -> Result<()> {
+        if util::is_chrome_only(&self.attributes) {
+            return Ok(())
+        }
+        if self.inheritance.is_some() {
+            warn!("skipping callback interface with inheritance: {}",
```

Comment:
Do we ever hit this in our webidls?

Reply:
Thankfully no! I made sure to check that before writing this :)